azure-devops/ecommerce/06_pagopa-ecommerce-helpdesk-service.tf

```diff
@@ -0,0 +1,130 @@
+  }
+}
+
+locals {
+  # global vars
+  pagopa-ecommerce-helpdesk-service-variables = {
+    cache_version_id = "v1"
+    default_branch   = var.pagopa-ecommerce-helpdesk-service.repository.branch_name
+  }
+  # global secrets
+  pagopa-ecommerce-helpdesk-service-variables_secret = {
+
+  }
+  # code_review vars
+  pagopa-ecommerce-helpdesk-service-variables_code_review = {
+    danger_github_api_token = "skip"
+    sonarcloud_service_conn = var.pagopa-ecommerce-helpdesk-service.pipeline.sonarcloud.service_connection
+    sonarcloud_org          = var.pagopa-ecommerce-helpdesk-service.pipeline.sonarcloud.org
+    sonarcloud_project_key  = var.pagopa-ecommerce-helpdesk-service.pipeline.sonarcloud.project_key
+    sonarcloud_project_name = var.pagopa-ecommerce-helpdesk-service.pipeline.sonarcloud.project_name
+  }
+  # code_review secrets
+  pagopa-ecommerce-helpdesk-service-variables_secret_code_review = {
+
+  }
+  # deploy vars
+  pagopa-ecommerce-helpdesk-service-variables_deploy = {
+    github_connection = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_github_rw_name
+
+    # acr section
+    k8s_image_repository_name            = replace(var.pagopa-ecommerce-helpdesk-service.repository.name, "-", "")
+    dev_container_registry_service_conn  = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_acr_aks_dev_id
+    dev_container_registry_name          = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_acr_aks_dev_name
+    uat_container_registry_service_conn  = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_acr_aks_uat_id
+    uat_container_registry_name          = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_acr_aks_uat_name
+    prod_container_registry_service_conn = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_acr_aks_prod_id
+    prod_container_registry_name         = data.terraform_remote_state.app.outputs.service_endpoint_azure_devops_acr_aks_prod_name
+
+    # aks section
+    dev_kubernetes_service_conn  = azuredevops_serviceendpoint_kubernetes.aks_dev.id
+    uat_kubernetes_service_conn  = azuredevops_serviceendpoint_kubernetes.aks_uat.id
+    prod_kubernetes_service_conn = azuredevops_serviceendpoint_kubernetes.aks_prod.id
+
+    dev_container_namespace  = "pagopadcommonacr.azurecr.io"
+    uat_container_namespace  = "pagopaucommonacr.azurecr.io"
+    prod_container_namespace = "pagopapcommonacr.azurecr.io"
+
+  }
+  # deploy secrets
+  pagopa-ecommerce-helpdesk-service-variables_secret_deploy = {
+    git_mail     = module.secrets.values["azure-devops-github-EMAIL"].value
+    git_username = module.secrets.values["azure-devops-github-USERNAME"].value
+    tenant_id    = module.secrets.values["TENANTID"].value
+  }
+}
+
+module "pagopa-ecommerce-helpdesk-service_code_review" {
+  source = "git::https://github.com/pagopa/azuredevops-tf-modules.git//azuredevops_build_definition_code_review?ref=v2.2.0"
```

Comment:
please use the last version of the module

Reply:
Left this module version for two reason:

1.  align module used by this ecommerce service to the other ones
2. latest version will not override trigger with this configuration

So I prefer to left this version in order to have the same behavior as for the other ecommerce services